### PR TITLE
Abstracts for bash, git and teams chapters

### PR DESCRIPTION
**Pull request by cwickham**
Just the chapters for which I'm responsible.
I'll sort out integrating this with @joelostblom once his have been merged.

### CODE REVIEW

**Comment by cwickham** [comment]
Ugh just realised I'm missing one...fixing now.

File: chapters/chapter-abstracts.md [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+One approach to combine programs 
+is to redirect the output from a shell command to a file,
+instead of printing it,
+then use the output file as the input file to another
+shell command.
+An alternative way to combine programs is the pipe,
+a way to link the output of one command to the input of
+another without intermediate files.
+A loop is a way to repeat a set of commands for each item in a list.
+By the end of the chapter a learner,
```
Comment (DamienIrving) [inline chapters/chapter-abstracts.md]:
```suggestion
By the end of the chapter a learner
```

File: chapters/chapter-abstracts.md [hunk 2]
```diff
@@ -0,0 +1,125 @@
...
+Instead of editing the code directory, 
+a working copy of the code is checked out, 
+edited, 
+then committed back to the repository. 
+A local repository is located the user's own computer.
+A remote repository is hosted on another computer or server, like GitHub. 
+By referring to specific Git commits, 
+previous versions of files can be viewed,
+or restored.
+Not every file’s history, needs to be tracked,
```
Comment (DamienIrving) [inline chapters/chapter-abstracts.md]:
```suggestion
Not every file’s history needs to be tracked,
```

File: chapters/chapter-abstracts.md [hunk 3]
```diff
@@ -0,0 +1,125 @@
...
+Two of Git's advanced features let us to do much more than just track our work.
+Branches let us work on multiple things simultaneously in a single repository;
+pull requests submit work for review,
+to get feedback,
+and to make updates.
+Used together,
+they allow a write-review-revise cycle
+familiar to anyone who has ever written a journal paper
+in hours rather than weeks.
+This chapter, presents a branch-per-feature workflow to develop new features while leaving the master branch in working order.
```
Comment (DamienIrving) [inline chapters/chapter-abstracts.md]:
```suggestion
This chapter presents a branch-per-feature workflow to develop new features while leaving the master branch in working order.
```

File: chapters/chapter-abstracts.md [hunk 4]
```diff
@@ -0,0 +1,125 @@
...
+By the end of the chapter a learner will be able to use the workflow in individual projects by,
```
Comment (DamienIrving) [inline chapters/chapter-abstracts.md]:
```suggestion
By the end of the chapter a learner will be able to use the workflow in individual projects by
```

File: chapters/chapter-abstracts.md [hunk 5]
```diff
@@ -0,0 +1,125 @@
...
+forking someone else's repository,
+and creating pull requests to submit changes.
+
+## 8. Working in Teams
+
+Projects can run for years with poorly-written code,
+but none will survive for long if people are confused,
+pulling in different directions,
+or hostile to each other.
+This chapter therefore looks at how to create a culture of collaboration
```
Comment (DamienIrving) [inline chapters/chapter-abstracts.md]:
```suggestion
This chapter looks at how to create a culture of collaboration
```

File: chapters/chapter-abstracts.md [hunk 6]
```diff
@@ -0,0 +1,125 @@
...
+that will help people who want to contribute to a project,
+and introduce a few ways to manage projects and teams as they develop.
+In particular the chapter suggests ways to
```
Comment (DamienIrving) [inline chapters/chapter-abstracts.md]:
```suggestion
In particular, the chapter suggests ways to
```

**Review by DamienIrving — APPROVED** [review]
Thanks, @cwickham. This is looking good - I've made a few very minor comments, but otherwise I'd say it's ready to merge.

You've done a good job sticking to writing in the third person, so @joelostblom can probably follow your lead when making edits to #588 to ensure consistency.

**Pull request merged**